apps/payments/next/app/[locale]/[offeringId]/[interval]/location/page.tsx

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {
+  BaseParams,
+  SelectTaxLocation,
+  buildRedirectUrl,
+} from '@fxa/payments/ui';
+import { config } from 'apps/payments/next/config';
+import { fetchCMSData } from '@fxa/payments/ui/actions';
+import { getApp, TermsAndPrivacy } from '@fxa/payments/ui/server';
+import { headers } from 'next/headers';
+import { redirect } from 'next/navigation';
+
+export const dynamic = 'force-dynamic';
+
+export default async function Location({
```

Comment:
Do we have any sort of logging to indicate how often this page is loaded?

From the jira ticket
> Log when this page is viewed so that we can keep track of how often a customer is running into this scenario

Reply:
d'oh